src/pages/components/button/usage.mdx

```diff
@@ -102,186 +132,419 @@ import { Add16 } from "@carbon/icons-react";
+
+#### Fluid versus fixed buttons
+
+Button alignment is also closely related to whether the button is treated as a fixed or a fluid element within a layout. When we say “fluid”, we mean that the button becomes a part of a larger, compound component by bleeding to two or more edges of its container. Rather than defining the a fluid button’s width in columns or mini units, its width is defined as a percentage (often 50%) of the container’s width. Also, as a general rule, fluid buttons are never left-aligned in a layout or a container — they’re always either right-aligned, or span the full width of the container.
+
+Fluid components like button never exist in isolation. As you can see in the examples above, they are always part of another component, like modal or form. Other fluid components include tiles and most recently, [text inputs](/components/text-input/usage).
 
 <Row>
 <Column colLg={8}>
 
-![Secondary / Primary button order with a third action in a modals](images/button-usage-5.png)
+![Example of a fluid input field and fluid button](images/button_usage_11.png)
 
 <Caption>
-  Secondary / Primary button pair with a tertiary action in a modal
+  Example of a login screen using a fluid input field and a fluid button
 </Caption>
 
 </Column>
 </Row>
 
-Use this same layout for buttons in side panels. Use full-width single buttons or full-width Secondary / Primary button pairs. If there is a third action, considering placing it within the side panel itself, rather than as part of the button group.
+<DoDontRow>
+  <DoDont caption="Do bleed ghost button hovers when they are paired with other fluid buttons.">
+
+![Do bleed ghost button hovers when they are paired with other fluid buttons.](images/button_usage_12.png)
 
-Side panel buttons should be visible without scrolling. They should be visible if they’re at the top, the bottom (without scrolling), or sticky and at the bottom of the view if the content is longer than the viewport.
+  </DoDont>
+  <DoDont type="dont" caption="Do not use a tertiary button in a fluid application.">
 
-### Non-modal forms
+![Do not use a tertiary button in a fluid application.](images/button_usage_13.png)
 
-On non-modal or in-page forms, align single buttons or Secondary / Primary button groups to the left. Buttons should align with the form controls regardless of the user‘s window width.
+  </DoDont>
+</DoDontRow>
+
+#### Fluid button border
+
+Many product designers have approached us looking for more guidance around borders between all fluid buttons. Although we haven’t made [this update](https://github.com/carbon-design-system/carbon/issues/5638) yet, we plan to add a 1px border between all fluid buttons that calls the \$ui-03 token for subtle borders. This feature will add a 3:1 distinction between the two interactive UI elements. The border is a recommended feature to improve accessibility in data visualizations and the same logic should apply here.
```

Comment:
Similar to the fluid button comment above, might help to defer on this section until we have a way to talk about this in design and code. The callout to the issue can be misleading because it's talking about adding behavior to fluid buttons, which is not something we currently offer and the linked issue isn't something that's on our immediate roadmap.

Reply:
This is an update that we're adding to modal, questions about 3:1 accessibility with modals buttons are all over our support channel, we need to notify our community about this update.